src/NuGet.Clients/NuGet.CommandLine/Commands/Pack/AssemblyMetadataExtractor.cs

```diff
@@ -96,7 +96,6 @@ public void ExtractMetadata(PackageBuilder builder, string assemblyPath)
             }
         }
 
-        [SuppressMessage("Microsoft.Performance", "CA1812:AvoidUninstantiatedInternalClasses", Justification = "It's constructed using CreateInstanceAndUnwrap in another app domain")]
```

Comment:
Was it false positive? 
Is latest VS becoming more smarter not to trigger this false positive?

Reply:
The code using MetadataExtractor changed.